MusicTaste/MusicTaste/View Controllers/DetailsViewController.m

```diff
@@ -174,14 +181,21 @@ - (BOOL)isRejectReached {
 }
 - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     if([[segue identifier] isEqualToString:@"backSegue"]) {
-        UINavigationController *navigationController = [segue destinationViewController];
-        MatchmakingViewController *matchmakingController = (MatchmakingViewController*)navigationController.topViewController;
-        //Passes the matches for the matchmakingViewController
-        matchmakingController.matches = self.matches;
+        UITabBarController *tabBar = [segue destinationViewController];
+        for (UIViewController *viewcontroller in tabBar.viewControllers) {
```

Comment:
Don't forget to use camelCase (`viewController`)

Reply:
will do!